examples/google-signin/google-signin-iframe.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <script src="https://accounts.google.com/gsi/client"></script>
+  <script async src="https://news.google.com/swg/js/v1/google_signin.js"></script>
+  <script>
+    // Add all allowed publisher domains.
+    const ALLOWED_ORIGINS = [
+      'http://localhost:8000',
+      'https://scenic-2017.appspot.com/'
+    ];
+    // Publisher's application Client ID (Google Developer Console)
+    const CLIENT_ID = '520465458218-embpp4s35v0utlr31317qkso3fk8q3d9.apps.googleusercontent.com';
+    // Publisher's login endpoint. Where Google Sign-in's response gets sent to.
+    // Must be able to handle a Google CredentialResponse within a FormData object:
+    // https://developers.google.com/identity/gsi/web/reference/js-reference#CredentialResponse
+    const LOGIN_ENDPOINT = 'https://scenic-2017.appspot.com/login';
+
+    // Callback which handles a Google Sign-in response. This callback should call a publisher
+    // endpoint and return a JSON Metering object described here:
+    // https://github.com/subscriptions-project/swg-js/blob/main/docs/entitlements-flow.md#swg-entitlements-flow
+    // The input argument `response` is a Google Sign-in CredentialResponse:
+    // https://developers.google.com/identity/gsi/web/reference/js-reference#CredentialResponse
+    const publisherGoogleSignInCallback = (response) => {
+      if (!response || !response['credential']) {
+        return;
+      }
+      // Creates a FormData object containing the response info.
+      const formData = new FormData();
+      Object.keys(response).map(function (name) {
+        formData.append(name, response[name]);
+      });
+      const init = /** @type {!./xhr.FetchInitDef} */ ({
+        method: 'POST',
+        credentials: 'include',
+        body: formData,
+      });
+      return window.fetch(LOGIN_ENDPOINT, init);
```

Comment:
IE11 doesn't support fetch

Reply:
Added in a note that IE doesn't support fetch. We are okay with publishers not supporting it if they don't want to.

Reply:
Can you expand the notes to explain that pub should persist the GIS response data, make an account if needed or return state for one that already exists, and set a cookie.  Regarding IE11 expand the not to indicate that if a pub wants to support IE11 they will need to use xhr